### DISCUSSION TITLE
[1.2.10] Updated ALI213 emus

### DISCUSSION THREAD

**Discussion by BigBoiCJ** [BigBoiCJ/SteamAutoCracker#32] [Announcements]
- Updated ALI213 Steam emulators for game cracking to v8.2.21.95 (#31, thanks to @v1ckxy)
  - Source for the files: [mafia47 on cs.rin.ru](https://cs.rin.ru/forum/viewtopic.php?p=2857168#p2857168)

NOTE: Deleting the config file isn't required.

SteamAutoCracker requires a 64 bits Windows computer.

<hr /><em>This discussion was created from the release <a href='https://github.com/BigBoiCJ/SteamAutoCracker/releases/tag/1.2.10'>[1.2.10] Updated ALI213 emus</a>.</em>

**Comment by BigBoiCJ** [comment]
🎉 By the way, SAC has reached 200 GitHub stars and 19000 downloads! Thanks everyone ❤️

**Reply by ashgithub2100** [reply]
do i have to download all of these 3 zips ?

**Reply by BigBoiCJ** [reply]
@ashgithub2100 No, only the one called Steam.Auto.Cracker.v1.2.10.zip
